python/lsst/ts/scheduler/driver/driver_target.py

```diff
@@ -27,6 +28,8 @@
 
 from lsst.ts.observatory.model import Target
 
```

Comment:
I don't think this empty line is necessary so please remove.

Reply:
it separates the `from lsst.ts` import from the `from .` import, for visual cue..